src/lib/with-traitify.js

```diff
@@ -219,11 +281,13 @@ export default function withTraitify(WrappedComponent) {
       if(ui && ui.options[name] != null) { return ui.options[name]; }
     }
     isReady = (type) => {
-      const {assessment, deck} = this.state;
+      const {assessment, deck, guide} = this.state;
 
       switch(type) {
         case "deck":
           return !!((deck && !!deck.name));
+        case "guide":
+          return !!((guide && (guide.competencies || []).length > 0));
```

Comment:
Why do you do this second check? It seems like if a guide were not null competencies would always be there